### DISCUSSION TITLE
docs/api/translate/baidu

### DISCUSSION THREAD

**Discussion by giscus** [STranslate/STranslate#106] [Announcements]
# docs/api/translate/baidu

WPF 开发的一款即开即用、即用即走的翻译(OCR)工具

https://stranslate.zggsong.com/docs/api/translate/baidu.html

**Comment by llllon** [comment]
具体如何调用百度翻译api？stranslate没有找到可以填入AppID 和密钥的配置输入框

**Reply by ZGGSONG** [reply]
添加服务方式参考 [本地服务](https://stranslate.zggsong.com/docs/api/translate/local.html)

![image](https://github.com/ZGGSONG/STranslate/assets/49741009/05bea757-d58a-4426-8e6b-b9d5e5959b35)
